src/charts/stacked-bar.js

```diff
@@ -572,41 +572,43 @@ define(function(require){
          * @return {obj}            Data entry that is closer to that x axis position
          */
         function getNearestDataPoint(mouseX) {
-            let adjustedMouseX = mouseX - margin.left,
-                dataByValueParsed = transformedData.map((item) => {
+            const adjustedMouseX = mouseX - margin.left;
+            const dataByValueParsed = transformedData.map((item) => {
                     item.key = item.key
```

Comment:
Hhmmm... not sure what are we doing here!

Reply:
I miss that, this is totally useless